client/components/select-dropdown/test/index.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+import { assert } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import SelectDropdown from '../index';
+
+describe( 'index', function() {
+	require( 'test/helpers/use-fake-dom' )();
```

Comment:
This is more of a Babel implementation detail at the moment, but `require` is synchronous while `import` is `async`. We generally only use require when importing dynamically (only in certain cases), or when we need to control the load order. For example mocking out packages when using use-mockery